pkg/Makefile.am

```diff
@@ -43,7 +43,9 @@ EXTRA_DIST += build.js files.js package.json package-lock.json
 
 # This is how the qunit tests get included.  We need to prevent automake from
 # seeing them during ./autogen.sh, but need make to find them at compile time.
+if WITH_COCKPIT_BRIDGE
```

Comment:
Can you please add a `FIXME:` comment here? The qunit tests are very much supposed to work against the python bridge, and we have relied on them heavily to ensure feature parity and for fixing bugs. I'm ok with temporarily disabling them to unblock this, but for me it would then be a #1 priority to get them back.

Reply:
I did this for two reasons:
 - we run the tests as part of pytest anyway
 - the tests don't work anyway since there's no `./cockpit-bridge` in the build directory for us to run.  Even when I tried to create it, something wasn't working right.  I didn't spend much more time trying to sort it out because, as written above, we run them as part of pytest anyway.